components/Workspaces/WorkspaceWelcomeModal.tsx

```diff
@@ -43,11 +43,17 @@ export default function WorkspaceWelcomeModal({ isOpen, onClose }: WorkspaceWelc
       title={TITLE}
       description={
         <div className="text-center flex flex-col items-center justify-between gap-4 !p-8">
-          <img
-            src="/assets/workspace_overview.png"
-            alt="Workspace screenshot from documentation"
+          <iframe
```

Comment:
nit: The iframe is the same for small screens and larger screens. Consider making it a constant chunk of JSX.